frontend/src/pages/FAQ.vue

```diff
@@ -355,8 +355,8 @@
               <template v-slot:mainnet>
                 <a :href="getEtherscanUrl(deployments.umbra, 1)" class="hyperlink" target="_blank"> mainnet </a>
               </template>
-              <template v-slot:goerli>
-                <a :href="getEtherscanUrl(deployments.umbra, 4)" class="hyperlink" target="_blank"> Goerli </a>
+              <template v-slot:sepolia>
+                <a :href="getEtherscanUrl(deployments.umbra, 11155111)" class="hyperlink" target="_blank"> Sepolia </a>
```

Comment:
We still support goerli, so reminder to add all that back to this FAQ page

Reply:
Added it back here: https://github.com/ScopeLift/umbra-protocol/pull/521/commits/6a191db4987fbe70868aa13b65e6d4db3d1e21e1